dashboard/src/main/resources/templates/component.ftl

```diff
@@ -3,6 +3,7 @@
   <head>
     <title>Google Cloud Platform Dependency Analysis Report for ${groupId}:${artifactId}:${version}</title>
     <link rel="stylesheet" type="text/css" href="dashboard.css" />
+    <script src="dashboard.js"></script>
```

Comment:
Google html style does not specify type attribute https://google.github.io/styleguide/htmlcssguide.html#type_Attributes